_vendor/github.com/gohugoio/gohugoioTheme/layouts/partials/social-follow.html

```diff
@@ -1,6 +1,6 @@
 <a
-  href="https://twitter.com/intent/follow?screen_name=gohugoiov2"
-  title="Follow on Twitter"
+  href="https://x.com/intent/follow?screen_name=gohugoiov2"
```

Comment:
The theme is vendored. Please make these changes in the https://github.com/gohugoio/gohugoioTheme repository.